app/views/site/index.html.erb

```diff
@@ -29,7 +29,11 @@
         Ensure any updates to this ID are reflected in the ReactDOM.render call
         in app/javascript/packs/application.js
       #%>
-      <div id="home-page-content"></div>
+      <% if User.feature_flag_for_instance(current_user, 'cross_dataset_search') && @selected_branding_group.nil? %>
```

Comment:
I think this is already showing we might need two feature flags -- one for XDS UX, and one for XDS backend.  Because we'll want the new XDS UX to work with branded spaces -- we just don't want it to then allow actual XDS search.

Reply:
I've split the one XDS flag into frontend and backend flags in bc3e5d3.